### ISSUE TITLE
HTML5 attributes: incorrect error message.

### ISSUE THREAD

**Issue by monnster** [Knockout-Contrib/Knockout-Validation#385]
Hi, i've a problem with HTML5 attributes, say min and max. When I set it to some value and try to enter incorrect numbers I get the following validation message:

```
Please enter a value less than or equal to [object Object].
```

A bit of ivestigation gets me to the validateSync function. In case validation rules are defined via code, not attributes, the ctx.params contains a number, but when the rule is defined with html5 attributes it contains the following object:

```
{ typeAttr: "number", value: 100 }
```

Which leads to the error message above.
